.dotnet/src/Utility/Polyfill/System.Runtime.CompilerServices.CompilerFeatureRequiredAttribute.cs

```diff
@@ -6,7 +6,7 @@ namespace System.Runtime.CompilerServices;
 internal sealed class CompilerFeatureRequiredAttribute(string featureName) : Attribute
 {
     public string FeatureName { get; } = featureName;
-    public bool IsOptional { get; init; }
+    public bool IsOptional { get; set; }
```

Comment:
This can be reverted since it's just an internal utility.

Reply:
Since we are fixing it and other internal attribute types have setters as well, let's keep the setter for it.